src/functions/severa/get-flextime-by-user/index.ts

```diff
@@ -0,0 +1,16 @@
+import { handlerPath } from "@libs/handler-resolver";
+
+export default {
+	handler: `${handlerPath(__dirname)}/handler.main`,
+  events: [
+    {
+      httpApi: {
+        method: "get",
+        path: "/severa/flextime/{severaGuid}",
```

Comment:
I would suggest to not use severaGuid and use severaUserGuid and severaProjectGuid. Although, I am not sure if it is not gonna get messy or not.